### PR TITLE
Added github to official links

### PR DESCRIPTION
**Pull request by g1nt0ki**

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -73,6 +73,7 @@
   * [Mine](https://mine.treasure.lol)
   * [Discord](https://discord.gg/treasuredao)
   * [Twitter](https://twitter.com/Treasure\_NFT)
+  * [Github] (https://github.com/TreasureProject)
```
Comment (karelvuong) [inline SUMMARY.md]: Small nit - could you update to "GitHub" with the capital H? Thanks for contributing!

**Review by karelvuong — CHANGES_REQUESTED** [review]
See my comment here!: https://github.com/TreasureProject/treasure-docs/pull/5#discussion_r820103665

File: SUMMARY.md (hunk 1 above)
Comment (g1nt0ki) [reply SUMMARY.md]: Hi, 

I made the fix. Since it was minor change, amended it instead of making a new commit.

**Comment by g1nt0ki** [comment]
@0xkarel I have made the fix, can you check?

**Pull request merged**